TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs

```diff
@@ -25,8 +25,8 @@ public void OnPost()
            
             using (var webClient = new WebClient())
             {
-                String countryJSON = webClient.DownloadString("http://api.nobelprize.org/v1/country.json");
-                    Country country = Country.FromJson(countryJSON);
+                String countryList = webClient.DownloadString("http://api.nobelprize.org/v1/country.json");
+                    Country country = Country.FromJson(countryList);
                     List<Count> random1 = country.Countries;
```

Comment:
I updated a couple of Variable names for better description and understanding.

Reply:
Actually, the variable country makes more sense to be countryList, as country indicates one country, but there are actually several.